install/kubernetes/cilium/values.yaml

```diff
@@ -2339,7 +2339,10 @@ tunnelPort: 0
 #  - drop
 serviceNoBackendResponse: reject
 
-# -- Configure the underlying network MTU to overwrite auto-detected MTU.
+# -- Configure the underlying network MTU to overwrite auto-detected MTU. 
+# This value doesn't change the host network interface MTU i.e. eth0 or ens0.
+# It changes the MTU for cilium_net@cilium_host, cilium_host@cilium_net,
+# cilium_vxlan and lxc_health interfaces.
 MTU: 0
```

Comment:
would it be helpful to add a prefix to MTU? like Cilium_MTU? some way to distinguish between cilium intface and host interface MTU?

Reply:
A neat solution would be, to do something like this:
```
mtu:
  cilium: 0
  host: 
    - ens0: 1500
```

Reply:
that would definitely make it very clear.

Reply:
True, I think the desired solution all depends on whether we will support changing the host interface MTU from within Cilium. If not, then the current PR will do everything that's needed.

Reply:
Can we resolve this discussion? Or move it somewhere? Its the only thing holding up the merging ATM.